snscrape/modules/twitter.py

```diff
@@ -32,7 +32,8 @@
 import typing
 import urllib.parse
 import warnings
-
+import orjson
+json = orjson
```

Comment:
Better to use `import orjson as json`, or better yet, replace all references to `json` with `orjson`.